app/services/acceptable_verified_credential_adoption_service.rb

```diff
@@ -77,7 +78,29 @@ def verified_credential_at?
     user_avc&.acceptable_verified_credential_at
   end
 
-  def log_results(conversion_type)
+  def log_conversion_type_results(conversion_type)
     StatsD.increment("#{STATS_KEY}.#{conversion_type}.#{credential_type}")
   end
+
+  def record_adoption_email_trigger_event
```

Comment:
ICN will always be available if the user is verified. However email may be blank. So there should probably be a check at the start of all this to 'do nothing' if `user.email` is blank

Reply:
Did not know that!  I'll make the change :)